### ISSUE TITLE
How to reduce or remove the string table?

### ISSUE THREAD

**Issue by weixiongmei** [NETMF/netmf-interpreter#338]
There's a StringTableData.cpp, it used a lot of space of flash, is it possible to remove or reduce the size of it? What's it for? Thank you~

**Comment by cw2** [comment]
Yes, it is possible - please have a look at issue #56 and the related pull request #284, its comment contains instructions how to rebuild the string table.

**Comment by weixiongmei** [comment]
hi cw2,

i got an error when running this command
msbuild Tools\Targets\Microsoft.SPOT.Support.Settings /t:buildstring

it shows '-loadstrings' is not an internal or external command, and it's not an excutable application.  Any idea? Thanks a lot~~~

**Comment by cw2** [comment]
You have to set PRG_MMP environment variable to metadataprocessor.exe path

```
set PRG_MMP=X:\NETMF\BuildOutput\ ??? \metadataprocessor.exe
```

**Comment by weixiongmei** [comment]
Thank you, it works~~~
